setup.py

```diff
@@ -99,6 +99,7 @@ def recursive_data_files(directory, install_directory):
 install_requires = [
     "attrs",
     "click",
+    "craft-application@git+https://github.com/canonical/craft-application@main",
```

Comment:
Snapcraft is going to need @cmatsuoka 's grammar integration (and thus craft-application 2.0), correct?

Reply:
Correct!